spec/index.bs

```diff
@@ -6659,19 +6697,23 @@ dictionary GPUQuerySetDescriptor : GPUObjectDescriptorBase {
 };
 </script>
 
-<dl dfn-type=dict-member dfn-for=GPUQuerySetDescriptor>
-    : <dfn>pipelineStatistics</dfn>
-    ::
-        The set of {{GPUPipelineStatisticName}} values in this sequence defines which pipeline statistics will be returned in the new query set.
+- {{GPUQuerySetDescriptor/type}} specifies the type of queries managed by {{GPUQuerySet}}.
+- {{GPUQuerySetDescriptor/count}} specifies the number of queries managed by {{GPUQuerySet}}.
+- {{GPUQuerySetDescriptor/pipelineStatistics}} is a set of {{GPUPipelineStatisticName}} values to
+    determine which pipeline statistics will be returned in {{GPUQuerySet}}.
```

Comment:
Please keep the `<dl dfn-type=dict-member dfn-for=GPUQuerySetDescriptor>` like before, and add the `type` and `count` to that definition list.